app/assets/stylesheets/chat.scss

```diff
@@ -60,100 +80,32 @@
   }
 }
 
-.chat__channels {
-  width: 45px;
-  min-width: 45px;
-  height: inherit;
-  position: relative;
-  overflow: hidden;
-  padding-top: 5px;
-
-  .chat__channelssearchtoggle {
-    background: transparent;
-    border: 0px;
-    margin-left: 2px;
-    svg {
-      vertical-align: -5px;
-      fill: var(--indicator-default-color);
-    }
-  }
-  input {
-    outline: 0;
+.chat__channels--placeholder {
+  .chat__channelstogglebutt--placeholderunexpanded {
+    display: none;
   }
-}
-
-.chat__channels--expanded {
-  width: 219px;
-  min-width: 219px;
 
-  @media screen and (min-width: 1000px) {
-    width: 200px;
-    min-width: 200px;
+  @media screen and (max-width: 599px) {
```

Comment:
This was already in here and I didn't make any changes to it. Because of the limitations of local development, I'm still not totally sure what this class covers, so this is something I'm not making updates on in this branch.